src/audio/CMakeLists.txt

```diff
@@ -95,6 +95,11 @@ if(NOT CONFIG_LIBRARY)
 	if(CONFIG_COMP_RTNR)
 		add_subdirectory(rtnr)
```

Comment:
Out of curiosity, what is basefw? Can you add some details in the commit message.

Reply:
sure. There are three types of elf binary in SOF:  boot_loader, basefw (built from all sof entities selected in kconfig) and external libraries (developed by 3rd party, may be close source). Rimage packs boot_loader and basefw into sof-xxx.ri